code/obj/table.dm

```diff
@@ -470,6 +470,19 @@
 		else
 			return ..()
 
+/obj/table/CanPass(atom/A, turf/T)
+	if (ismonkey(A)) // You Shall Not Pass!
+		var/mob/living/M = A
+		if(M.lying) // Lying down monkeys can crawl under tables
+			M.layer = src.layer - 0.01
+			return 1
+		else
+			M.layer = MOB_LAYER
+			return 0
+	else
+		return 0
+	return ..()
```

Comment:
This line will never be called because all code paths hit a `return` first. I don't know if that (i.e. not calling `CanPass` on `/obj`) is a problem (or if it even exists), but would check interaction with critters here to make sure this doesn't stop them from crawling over/under.

Reply:
Oh, aside, this is why the linter is failing.